Riot/Modules/TabBar/MasterTabBarController.m

```diff
@@ -459,164 +459,52 @@ - (void)onMatrixSessionStateDidChange:(NSNotification *)notif
-//        MXStrongifyAndReturnIfNil(self);
-//
-//        // What was this doing? This should probably happen elsewhere
-//        // self.onboardingCoordinatorBridgePresenter = nil;
-//        
-//        [[NSNotificationCenter defaultCenter] removeObserver:self.addAccountObserver];
-//        self.addAccountObserver = nil;
-//    }];
-//    
-//    self.removeAccountObserver = [[NSNotificationCenter defaultCenter] addObserverForName:kMXKAccountManagerDidRemoveAccountNotification object:nil queue:[NSOperationQueue mainQueue] usingBlock:^(NSNotification *notif) {
-//        MXStrongifyAndReturnIfNil(self);
-//        // The user has cleared data for their soft logged out account
-//
-//        // What was this doing? This should probably happen elsewhere
-//        // self.onboardingCoordinatorBridgePresenter = nil;
-//        
-//        [[NSNotificationCenter defaultCenter] removeObserver:self.removeAccountObserver];
-//        self.removeAccountObserver = nil;
-//    }];
-=======
-    MXLogDebug(@"[MasterTabBarController] presentOnboardingFlow");
-    
-    MXWeakify(self);
-    OnboardingCoordinatorBridgePresenter *onboardingCoordinatorBridgePresenter = [[OnboardingCoordinatorBridgePresenter alloc] init];
-    onboardingCoordinatorBridgePresenter.completion = ^{
-        MXStrongifyAndReturnIfNil(self);
-        [self.onboardingCoordinatorBridgePresenter dismissWithAnimated:YES completion:nil];
-        self.onboardingCoordinatorBridgePresenter = nil;
-        
-        self.isOnboardingInProgress = NO;   // Must be set before calling didCompleteAuthentication
-        [self.masterTabBarDelegate masterTabBarControllerDidCompleteAuthentication:self];
-    };
-    
-    [onboardingCoordinatorBridgePresenter presentFrom:self animated:NO];
-    
-    self.onboardingCoordinatorBridgePresenter = onboardingCoordinatorBridgePresenter;
-    self.isOnboardingCoordinatorPreparing = NO;
->>>>>>> v1.8.20
 }
 
 - (void)showOnboardingFlow
 {
-<<<<<<< HEAD
-    MXLogDebug(@"[MasterTabBarController] showAuthenticationScreen");
-    
-    // Check whether an authentication screen is not already shown or preparing
-    // Tchap: Disable Onboarding.
-//    if (!self.onboardingCoordinatorBridgePresenter && !self.isOnboardingCoordinatorPreparing)
-//    {
-//        self.isOnboardingCoordinatorPreparing = YES;
-//        self.isOnboardingInProgress = YES;
-        
-        [self resetReviewSessionsFlags];
-        
-        [[AppDelegate theDelegate] restoreInitialDisplay:^{
-                        
-//            [self presentOnboardingFlow];
-            [self.masterTabBarDelegate masterTabBarControllerShouldShowAuthenticationFlow:self];
-        }];
-//    }
-=======
     MXLogDebug(@"[MasterTabBarController] showOnboardingFlow");
     [self showOnboardingFlowAndResetSessionFlags:YES];
```

Comment:
Onboarding was disable in Tchap. Is this still the case ?

Reply:
The onboarding from Element is still disabled in Tchap, but I need to update the Tchap onboarding to be compliant to the last changes in Element.